packages/breez_sdk_liquid/lib/src/breez_sdk_liquid.dart

```diff
@@ -121,6 +121,7 @@ class BreezSDKLiquid {
         } else if (event is liquid_sdk.SdkEvent_PaymentFailed) {
           _paymentEventStream.addError(event);
         } else if (event is liquid_sdk.SdkEvent_Synced) {
+          await _listPayments(sdk: _instance!);
           _didCompleteInitialSyncController.add(null);
         }
         await _fetchWalletData(sdk);
```

Comment:
Isn't this also done in `_fetchWalletData(sdk)`?

Reply:
Done.
- Fire `didCompleteInitialSync` event after wallet data is fetched.